packages/gamut/src/Form/Checkbox.tsx

```diff
@@ -22,107 +28,84 @@ export type CheckboxProps = InputHTMLAttributes<HTMLInputElement> &
     id?: string;
   };
 
-type Multiline = { multiline?: boolean };
-
-const activeColor = colors.hyper;
+export type CheckboxTextProps = StyleProps<typeof checkboxTextStates>;
 
-const CheckboxLabel = styled.label`
-  ${noSelect}
-  display: flex;
-  align-items: flex-start;
-  cursor: pointer;
-  margin: (${variables.formPadding} / 2) 0;
-  width: 100%;
-  font-weight: normal;
-  padding: ${variables.formPadding} 0;
-`;
+const CheckboxLabel = styled.label(noSelect, checkboxLabel);
 
-const CheckboxElement = styled.div<Multiline>`
-  position: relative;
-  margin-right: 0.5rem;
-  min-width: 1.5rem;
-  height: 1.5rem;
-  border: 2px solid ${variables.borderColor};
-  border-radius: ${variables.borderRadius};
-  transition: all 0.1s ease-in-out;
-  margin-top: ${({ multiline }) => multiline && '3px'};
-`;
+const CheckboxElement = styled.div<Multiline>(
+  states({
+    multiline: {
+      mt: 4,
+    },
+  }),
+  checkboxElement
+);
```

Comment:
For predictability I like to order styles progressively in terms of specificity:
1. Base styles
2. Variants
3. States
4. Props

That way we can progressively override / customize behavior in a standardized way :) 

```suggestion
const CheckboxElement = styled.div<Multiline>(
  checkboxElement,
  states({
    multiline: {
      mt: 4,
    },
  }),
);
```